src/Conf.ml

```diff
@@ -466,7 +466,13 @@ let info =
          for each input file are used, as well as the global .ocamlformat \
          file defined in $(b,\\$XDG_CONFIG_HOME/ocamlformat). The global \
          .ocamlformat file has the lowest priority, then the closer the \
-         directory is to the processed file, the higher the priority." ]
+         directory is to the processed file, the higher the priority."
+    ; `P
+        "An `.ocamlformat-ignore` file specifies files that OCamlFormat \
+         should ignore.  Each line in an ocamlformat-ignore file specifies \
+         a filename relative to the dir of the ocamlformat-ignore file. \
+         Lines starting with `#` are ignored and can be used as comments."
```

Comment:
Same rewording as in the readme above. Also note that markdown isn't interpreted here, it is in groff format for man pages.